check_catkin_lint.sh

```diff
@@ -1,21 +0,0 @@
-# Software License Agreement - BSD 3-Clause License
-#
```

Comment:
We should have a look at [ament_lint](https://github.com/ament/ament_lint).

Reply:
> This should do good for a first prototype. I realize that my changes are probably out of the scope of this PR and not really necessary yet but it would be great if you could add corresponding TODOs.

In this case I would approving with a note that you only approve after I've added the appropriate TODO's. If you have comments, leave a comment. Requesting changes should be reserved for when you find issue with the code as it is. The red +/- keeps other reviewers from taking a look and essentially veto's the change until you've had a chance to look at the code again. In the case of a few small TODO's, this seems like overkill.